pkg/controller/jenkins/reconciler.go

```diff
@@ -117,32 +121,49 @@ func (r *JenkinsReconciler) Reconcile(request reconcile.Request) (reconcile.Resu
 	}
 	jenkinsSvc := newJenkinsService(instance, jenkinsInstanceName, jenkinsPort)                                  // jenkins service
 	jenkinsJNLPSvc := newJenkinsService(instance, jenkinsInstanceName+JenkinsJnlpServiceSuffix, jenkinsJNLPPort) // jenknis jnlp service
-	jenkinsPvc := newJenkinsPvc(instance, jenkinsInstanceName)                                                   // jenknis pvc
+	jenkinsRoute := newJenkinsRoute(instance, jenkinsSvc)
+	jenkinsPvc := newJenkinsPvc(instance, jenkinsInstanceName) // jenknis pvc
+	jenkinsServiceAccount := newJenkinsServiceAccount(instance, jenkinsInstanceName)
+	jenkinsRoleBinding := newJenkinsRoleBinding(instance, jenkinsInstanceName)
 
 	// Set Jenkins instance as the owner and controller
-	if err := controllerutil.SetControllerReference(instance, dc, r.scheme); err != nil {
+	if err := controllerutil.SetControllerReference(instance, jenkinsDc, r.scheme); err != nil {
```

Comment:
Not relevant to this PR, but you should add `ownerReferences` to all objects being created.

Reply:
done